scripts/create-darwin-volume.sh

```diff
@@ -11,7 +11,7 @@ root_disk_identifier() {
 }
 
 find_nix_volume() {
-    diskutil apfs list -plist "$1" | xmllint --xpath "/plist/dict/array/dict/key[text()='Volumes']/following-sibling::array/dict/key[text()='Name']/following-sibling::string[contains(translate(text(),'N','n'),'nix')]/text()" -
+    diskutil apfs list -plist "$1" | xmllint --xpath "/plist/dict/array/dict/key[text()='Volumes']/following-sibling::array/dict/key[text()='Name']/following-sibling::string[contains(translate(text(),'N','n'),'nix')]/text()" - || true 2>/dev/null
```

Comment:
Should this be `- 2>/dev/null || true`, not `- || true 2>/dev/null`?

```suggestion
    diskutil apfs list -plist "$1" | xmllint --xpath "/plist/dict/array/dict/key[text()='Volumes']/following-sibling::array/dict/key[text()='Name']/following-sibling::string[contains(translate(text(),'N','n'),'nix')]/text()" - 2>/dev/null || true
```

Reply:
Thanks :)